.github/workflows/go.yml

```diff
@@ -0,0 +1,50 @@
+name: continuous-integration
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
I think we should do this test on a few OS's - such as ad CentOS and others. We can not assume the end-user will always use Ubuntu. I think Github action allows you to run same workflow for a combination of OS - I would suggest exploring that.